pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dto/dashboard/DashboardDto.java

```diff
@@ -0,0 +1,84 @@
+package org.broadinstitute.dsm.db.dto.dashboard;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import lombok.Data;
+import org.broadinstitute.dsm.model.dashboard.DisplayType;
+import org.broadinstitute.dsm.model.dashboard.Size;
+
+@Data
+public class DashboardDto {
+
+    Integer dashboardId;
+    Integer ddpInstanceId;
+    String displayText;
+    DisplayType displayType;
+    Size size;
+    Integer order;
+    List<DashboardLabelDto> labels;
+
+    private DashboardDto(Builder builder) {
+        this.dashboardId = builder.dashboardId;
+        this.ddpInstanceId = builder.ddpInstanceId;
+        this.displayText = builder.displayText;
+        this.displayType = builder.displayType;
+        this.labels = builder.labels;
+        this.size = builder.size;
+        this.order = builder.order;
+    }
+
+    public List<String> getColors() {
+        return labels.stream().map(DashboardLabelDto::getColor).collect(Collectors.toList());
+    }
+
+    public static class Builder {
+
```

Comment:
Is there any reason to use traditional approach like this instead of using lomboks @Builder annotation?

Reply:
technically no, but I prefer the way of having builder functions with prefix `with` and `lombok` does not provide such prefixes as far as I know.